Enterprise-CMCS/macpro-mdct-mfp
# [MDCT-2731] - Modals

### Description


https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/19439679/753beb21-e36a-4fc7-82d4-365507bc241b



### Related ticket(s)


MDCT-2731

---
### How to test

I've included an example on the homepage for you to check it out and test. I'll remove it once this PR is deemed worthy! Please note that the add/edit modals will not submit the form due to validation not being implemented.

---
### Author checklist
